src/WebJobs.Extensions.Http/Routing/WebJobsRouter.cs

```diff
@@ -79,19 +73,27 @@ public void AddFunctionRoutes(IRouter functionRoutes, IRouter proxyRoutes)
             }
         }
 
+        public void AddCustomRoutes(IRouter customRoutes)
+        {
+            if (customRoutes != null)
+            {
+                _routeCollection.Add(customRoutes);
+            }
+        }
+
         public string GetFunctionRouteTemplate(string functionName)
         {
             if (functionName == null)
             {
                 throw new ArgumentNullException(nameof(functionName));
             }
 
-            if (_functionRoutes == null)
+            if (_routeCollection == null)
```

Comment:
Is this change just to ensure we're also processing the new/custom routes? The impact is that proxy routes will also be evaluated

Reply:
This method was just added for tests and I needed to update it for new custom route tests

---
In reply to: [511065642](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/668#discussion_r511065642) [](ancestors = 511065642)